core/src/test/java/com/valtech/aem/saas/core/fulltextsearch/SearchModelImplTest.java

```diff
@@ -106,15 +105,15 @@ void testAdaptResource() {
   }
 
   private void adaptRequest() {
-    testee = context.request().adaptTo(SearchModel.class);
+    testee = context.request().adaptTo(SearchModelImpl.class);
```

Comment:
why do we need to change to the impl? just add it to the models in the setup: 
context.addModelsForClasses( ... )

Reply:
This change is needed in order to test the SearchModelImpl:: getSearchTabs, which is not part of the SearchModel interface

Reply:
mmh, this should be part of the interface as well

Reply:
I don't see we should demand implementation of the method, when it is only needed for our implementation of the search component (it is used to generate the **searchTabs: []** portion of the search config json)